### PR TITLE
Fix multisig initialization

### PR DESCRIPTION
**Pull request by jleni**

### CODE REVIEW

File: src/lib/parser_txdef.c [hunk 1]
```diff
@@ -90,9 +90,11 @@ void parser_txInit(parser_tx_t *tx) {
...
 
+    tx->multisigPtr = NULL;
+    tx->multisigLen = 0;
+    parser_multisigInit(&tx->multisig);
+
     tx->sendmsgPtr = NULL;
     tx->sendmsgLen = 0;
     parser_sendmsgInit(&tx->sendmsg);
-
-    parser_multisigInit(&tx->multisig);
```
Comment (webmaster128) [inline src/lib/parser_txdef.c:97]: Just a question: why did you change the order? What is `tx->sendmsg`?

**Review by webmaster128 — APPROVED** [review]

File: src/lib/parser_txdef.c (hunk 1 above)
Comment (jleni) [reply src/lib/parser_txdef.c:97]: Just to keep the same order as in the struct definition.
It is not a functional change
